### DISCUSSION TITLE
IH-K009 - Tuya Temp/Humidity

### DISCUSSION THREAD

**Discussion by CreativeImpulse** [Koenkk/zigbee2mqtt#14224] [Devices]
I'm new to this...

I have a AliExpress purchase that claims to be a IH-K009 Tuya sensor. My z2m has responded:

  0xa4c13836788ab3f0      0xa4c13836788ab3f0 (0x8E37)    Unsupported

This to me suggests either it isn't what it claims to be or I need to do something to make it work. The device is in the list of supported devices.  I paired it by pressing the button for 5 seconds. The light flashed, etc. Just didn't hook up.

Any suggestions?

Rob

**Comment by CreativeImpulse** [comment]
Please ignore....  following more persistent pairing attempts it has connected up ok.

**Reply by pedronunesbr** [reply]
The same issue from the initial post was solved by trying again (reconfigure button on Zigbeee2Mqtt -> press reset button on the device). 
Thank you.
